ingestion/src/metadata/ingestion/source/database/database_service.py

```diff
@@ -309,6 +307,14 @@ def yield_view_lineage(self) -> Optional[Iterable[AddLineageRequest]]:
         Parses view definition to get lineage information
         """
 
+    def yield_table_constraints(self) -> Optional[Iterable[OMetaTableConstraints]]:
```

Comment:
Does this needs to be an abstractmethod?

Reply:
I prefer not to make this an abstract field because the non sqa sources does not support table constraints and if I make it an abstract method then for all non sqa sources I'll have to overwrite it with empty method, so I thought it would be better if I create an empty method at parent itself

Reply:
I see, do we need to have it here as an empty method at all or can we just implement it where it is supported?